Makefile

```diff
@@ -195,10 +195,18 @@ docker-push-argo:
 	$(MAKE) -C argo/kfp-compiler docker-push
 	$(MAKE) -C argo/providers docker-push
 
-##@ Website
+##@ Docs
 website:
 	$(MAKE) -C docs-gen
 
+docker-build-quickstart:
+	$(MAKE) -C docs-gen/includes/quickstart docker-build
```

Comment:
This is not needed anymore since we removed it from the PR build (thanks to the tiny TFX image).